CMakeLists.txt

```diff
@@ -236,96 +217,120 @@ set(COMMON src/common/logging/backend.cpp
          src/core/loader.h
+         src/core/loader/elf.cpp
+         src/core/loader/elf.h
+         src/core/loader/symbols_resolver.h
+         src/core/loader/symbols_resolver.cpp
+         src/core/libraries/error_codes.h  
+         src/core/libraries/libs.h
+         src/core/libraries/libs.cpp
+         ${AUDIO_LIB}
+         ${GNM_LIB}
+         ${KERNEL_LIB}
+         ${NETWORK_LIBS}
+         ${SYSTEM_LIBS}
+         ${LIBC_SOURCES}
+         ${PAD_LIB}
+         src/core/linker.cpp
+         src/core/linker.h
+         src/core/tls.cpp
+         src/core/tls.h     
+         src/core/virtual_memory.cpp
+         src/core/virtual_memory.h
 )
 
-set(CRYPTO src/core/crypto/crypto.cpp
-           src/core/crypto/crypto.h 
-           src/core/crypto/keys.h
+set(VIDEO_CORE src/core/PS4/HLE/Graphics/video_out.cpp
+               src/core/PS4/HLE/Graphics/video_out.h
+               src/core/PS4/HLE/Graphics/Objects/video_out_ctx.cpp
+               src/core/PS4/HLE/Graphics/Objects/video_out_ctx.h
+               src/core/PS4/HLE/Graphics/graphics_ctx.h
+               src/core/PS4/GPU/gpu_memory.cpp
+               src/core/PS4/GPU/gpu_memory.h
+               src/core/PS4/GPU/video_out_buffer.cpp
+               src/core/PS4/GPU/video_out_buffer.h
+               src/core/PS4/HLE/Graphics/graphics_render.cpp
+               src/core/PS4/HLE/Graphics/graphics_render.h
+               src/core/PS4/GPU/tile_manager.cpp
+               src/core/PS4/GPU/tile_manager.h
+               src/vulkan_util.cpp
+               src/vulkan_util.h
 )
-set(FILE_FORMAT src/core/file_format/pfs.h
-                src/core/file_format/pkg.cpp
-                src/core/file_format/pkg.h
-                src/core/file_format/pkg_type.cpp
-                src/core/file_format/pkg_type.h
-                src/core/file_format/psf.cpp
-                src/core/file_format/psf.h
+
+set(INPUT src/input/controller.cpp
+          src/input/controller.h
 )
 
+# the above is shared in sdl and qt version (TODO share them all)
+
+if(ENABLE_QT_GUI)
+qt_add_resources(RESOURCE_FILES src/shadps4.qrc)
+    set(QT_GUI 
+    src/qt_gui/main_window_ui.h
+    src/qt_gui/main_window.cpp
+    src/qt_gui/main_window.h
+    src/qt_gui/gui_settings.cpp
+    src/qt_gui/gui_settings.h
+    src/qt_gui/gui_save.h
+    src/qt_gui/gui_context_menus.h
+    src/qt_gui/game_list_utils.h
+    src/qt_gui/game_info.cpp
+    src/qt_gui/game_info.h
+    src/qt_gui/game_list_frame.cpp
+    src/qt_gui/game_list_frame.h
+    src/qt_gui/game_grid_frame.cpp
+    src/qt_gui/game_grid_frame.h
+    src/qt_gui/game_install_dialog.cpp
+    src/qt_gui/game_install_dialog.h
+    src/qt_gui/pkg_viewer.cpp
+    src/qt_gui/pkg_viewer.h
+    src/qt_gui/settings.cpp
+    src/qt_gui/settings.h
+    src/qt_gui/main_window_themes.cpp
+    src/qt_gui/main_window_themes.h
+    src/qt_gui/main.cpp
+    ${RESOURCE_FILES}
+    )
+endif()
+
 if(ENABLE_QT_GUI)
 qt_add_executable(shadps4
+    ${AUDIO_CORE}
```

Comment:
Audio core requires SDL, which I think is not compiled when qt is enabled need

Reply:
we have fixed that earlier :D

Reply:
gotcha